### PR TITLE
Fix PG10 beta2 break on predicate_refuted_by function call

### PR DESCRIPTION
**Pull request by mtuncer**
PG10 Beta2 adds ```clause_is_check``` parameter to predicate_refuted_by function.

### CODE REVIEW

File: cstore_reader.c [hunk 1]
```diff
@@ -789,7 +789,11 @@ SelectedBlockMask(StripeSkipList *stripeSkipList, List *projectedColumnList,
 							 blockSkipNode->maximumValue);
 
 			constraintList = list_make1(baseConstraint);
+#if (PG_VERSION_NUM >= 100000)
```
Comment (pykello) [inline cstore_reader.c]: Can we add PG10 to travis.yaml so this is automatically tested?

Comment (mtuncer) [inline cstore_reader.c]: Last time I could not get it to work. It looks like there has been some changes in tools project. Trying again.

**Comment by pykello** [comment]
Looks good. Merging.

**Comment by acscott** [comment]
Cool!

**Pull request merged**